cg/store/api/find_business_data.py

```diff
@@ -485,21 +487,34 @@ def get_samples_by_customer_id_and_pattern(
             filter_functions=filter_functions,
         ).all()
 
+    def get_samples_by_customer_and_subject_id_query(
```

Comment:
```suggestion
    def get_sample_query_by_customer_and_subject_id(
```

Reply:
this is on purpose since this function is used in two othe rfunctions and reuiqres a query output to work.